page_creation.rst

```diff
@@ -94,6 +94,9 @@ PHP versions you can use annotations. To do this, install the annotations packag
 .. code-block:: terminal
 
     $ composer require annotations
+    
```

Comment:
imho, starting from v6, we could no more document annotations things for feature implemented via native attributes 

Wdyt?

Reply:
Oh sorry i did not catch up!

I'll close pull request then
++